functions/commentPost/index.js

```diff
@@ -0,0 +1,33 @@
+import λ from 'apex.js'
```

Comment:
There's no APIDoc for this. Please add documentation so it is obvious how to use this endpoint.